tools/PI/DevHome.PI/Views/PrimaryWindow.xaml.cs

```diff
@@ -30,6 +30,8 @@ public PrimaryWindow()
     {
         InitializeComponent();
         ExternalToolsHelper.Instance.Init();
+        _hotKeyHelper = new(this, HandleHotKey);

```

Comment:
>_hotKeyHelper = new(this, HandleHotKey); [](http://example.com/codeflow?start=8&length=40)

Was moving this here required? Whitechapel had mentioned in the past that it seemed unreliable if we actually had an HWND in the constructor, which is why this was in the WindowLoaded callback.

Reply:
I will double check that we actually get the Window_Loaded callback in all cases

Reply:
I'll try moving it back and make sure the scenario still works

Reply:
Seems to work fine in  Window_loaded so I've moved it back.